bridgebot/scoring.py

```diff
@@ -312,3 +312,22 @@ def score(bid, strain, result, doubled, vulnerability):
     else:
         return no_trump(bid, result, doubled, vulnerability)
 
+
```

Comment:
I tried to keep this code as simple as possible, but it is called from `main.py`

Reply:
The ugly enum processing is encapsulated here, the game mechanic (i.e. which formula to use to calculate the score) is encapsulated in `calculate_score`, and the actual math is encapsulated in `major`, `minor` and `no_trump`